priv/update_swagger_url

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env escript
+
+-module(update_swagger_url).
+
+-export([main/1]).
```

Comment:
These are not needed and can be removed.

Reply:
Done!